Sources/UIExtensions/Components/SwiftUI/UIKit Wrappers/ShareSheetView.swift

```diff
@@ -0,0 +1,81 @@
+//
```

Comment:
Could you add a view extension that either uses a sheet or a popover (on the receiving view) depending on OS?

Reply:
If I understand your suggestion correctly, on iPad shows as popover, on iPhone as sheet? If this is the case, using popover will already give that for free.
If this is about mac/tvOS I am not sure if they have Share Sheet, maybe the mac, I have to check.

Reply:
I see, that’s good then :)